l10n_ro_account_report/views/report_account_bank_statement.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+<data>
+<template id="report_account_bank_statement2">
+  <t t-call="report.html_container">
+  <t t-foreach="docs" t-as="o">
+
+  <!--t t-call="report.internal_layout"-->
+
+<!--  <t t-set="data_report_margin_top" t-value="12"/>
+  <t t-set="data_report_margin_bottom" t-value="12"/>
+  <t t-set="data_report_header_spacing" t-value="8"/>
+-->    
+  <t t-call="l10n_ro_account_report.romanian_layout">
+        
+    <div class="page">
+      <h5 class="text-left">Societatea comerciala: <b><span t-esc="res_company.name"></span></b></h5>
+      <h5 class="text-left">Numar Registu Comertului <b><span t-esc="res_company.company_registry"></span></b></h5>
+      <h5 class="text-left">Cod fiscal: <b><span t-esc="res_company.vat"></span></b></h5>
+      <h5 class="text-left">Adresa: <span t-field="res_company.partner_id.street"/><span t-field="res_company.partner_id.street2"/> <span t-field="res_company.partner_id.city"/> <span t-field="res_company.partner_id.state_id"/></h5>
+      <h4  class="text-center bold"> Registru <span t-field="o.journal_id"/> </h4>
+      <div class="oe_structure" />
+        <h5 class="text-left">Data <b><span t-esc="o.date"></span></b></h5>
+        <h5 class="text-left">Perioada <b><span t-esc="o.period_id.name"></span></b></h5>
+        <table class="table table-condensed">
+          <thead>
+            <th  class="text-center">Nr Crt</th>
+            <th  class="text-center">Data</th>
+            <th  class="text-center">Partener</th>
+            <th  class="text-center">Explicatia</th>
+            <th  class="text-center">Doc. casa</th>
+            <th  class="text-center">Tip</th>
+            <th  class="text-center">Incasari</th>
+            <th  class="text-center">Plati</th>
+            <th  class="text-center">Sold</th>
+          </thead>
+          <tbody>
+            <tr>
+              <td class="text-left" colspan="8"> <b >Sold initial: </b></td>
+              <td class="text-right"> <b><span  t-esc="o['balance_start']"></span></b>
+              </td></tr>
+            <t t-set="sold" t-value="o['balance_start']"/>
+            <tr t-foreach="o['line_ids']" t-as="l">
+            <t t-set="sold" t-value="sold+l.amount"/>
+<!--aici obiectul este account.bank.statement.line -->
+            <td><span t-esc="l_index + 1"/></td>
+            <td><span t-esc="l.date"></span><!--span t-esc="formatLang(l.date,date=True)"/--></td>                                     
```

Comment:
Raportul este/ poate fi si registru de banca.
La banca poti sa faci un extras pe mai multe zile- in general banca o proceseaza la cateva zile.